service/src/main/java/uk/gov/hmcts/reform/fpl/service/noc/NoticeOfChangeFieldPopulator.java

```diff
@@ -64,6 +65,30 @@ public Map<String, Object> generate(CaseData caseData, Representing representing
         return data;
     }
 
+    public Map<String, Object> generateApplicantAnswer(CaseData caseData) {
+        Map<String, Object> data = new HashMap<>();
+        NoticeOfChangeAnswers nocAnswers = NoticeOfChangeAnswers.builder().build();
+
+        if (caseData.isOutsourced()) {
+            LocalAuthority localAuthority = caseData.getLocalAuthorities().stream()
+                .map(Element::getValue)
+                .filter(la -> la.getId().equals(caseData.getOutsourcingPolicy().getOrganisation().getOrganisationID()))
+                .findFirst().orElse(null);
```

Comment:
findFirst() returns an optional - so could potentially use that instead of the explicit null check after